Submitted By: Jim Mischel <jim@mischel.com>
Date: 2003-09-11
Initial Package Version: 1.10
Origin: Jim Mischel <jim@mischel.com>
Description: This patch modifies the makefile and one source file of the SVGATextMode
             source package so that it will compile with GCC versions 3.2 and later.

--- SVGATextMode-1.10/Makefile	2000-09-02 18:43:36.000000000 -0500
+++ SVGATextMode-1.10.new/Makefile	2002-09-10 22:52:43.000000000 -0500
@@ -26,8 +26,8 @@
 CFLAGS_alpha =
 CFLAGS_i386  = -Wpointer-arith -Wnested-externs 
 CFLAGS_i486  =
-CFLAGS_i586  = -fno-strength-reduce -malign-loops=2 -malign-jumps=2 -malign-functions=2
-CFLAGS_i686  = -fno-strength-reduce -malign-loops=2 -malign-jumps=2 -malign-functions=2
+CFLAGS_i586  = -fno-strength-reduce -falign-loops=2 -falign-jumps=2 -falign-functions=2
+CFLAGS_i686  = -fno-strength-reduce -falign-loops=2 -falign-jumps=2 -falign-functions=2
 
 LDFLAGS_alpha =
 LDFLAGS_i386  =
--- SVGATextMode-1.10/cfgfile.y	2000-06-21 21:24:10.000000000 -0500
+++ SVGATextMode-1.10.new/cfgfile.y	2002-09-10 22:57:48.000000000 -0500
@@ -32,6 +32,7 @@
 #include "messages.h"
 #include "misc.h"
 #include <sys/stat.h>
+#include <stdio.h>
 
 /* if VGA_CAN_DO_64KB is defined, 32k chars per screen are allowed */
 #ifdef VGA_CAN_DO_64KB
